src/components/TabsPage.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  setSelectedTab: (id: Tab) => void,
+  selectedTabId: string,
+};
+
+export const TabsPage: FC<Props> = ({
+  tabs, setSelectedTab, selectedTabId,
+}) => {
+  const { tabId = 'tab-0' } = useParams();
```

Comment:
do not hardcode the value, better to use it from tabs
```suggestion
  const { tabId = tabs[0].id } = useParams();
```